Makefile

```diff
@@ -140,7 +141,7 @@ protoc-gen-all: protoc-gen-code protoc-gen-docs
 # Generate protobuf go code
 protoc-gen-code:
 	@protoc \
-		--go_out=plugins=grpc:./rpc \
+		--go_out=./rpc \
 		--go_opt=module=github.com/stripe/stripe-cli/rpc \
```

Comment:
This works when I modify a proto message and rerun `make protoc`. But when I add a new rpc to commands.proto, the compiler doesn't see it.

This combo of flags works for both, try this:

```
		--go_out=./rpc \
		--go_opt=module=github.com/stripe/stripe-cli/rpc \
		--go-grpc_out=require_unimplemented_servers=false:./rpc \
		--go-grpc_opt=module=github.com/stripe/stripe-cli/rpc \
```

Reply:
Fixed!